### DISCUSSION TITLE
Force Dark Mode on Xiaomi

### DISCUSSION THREAD

**Discussion by thproflord** [dofire/Floating-Bubble-View#64] [General]
So I have been doing tests, and having a Redmi Note 10S with dark mode turn on, the black texts are converted to white, is there any way of forcing to not changed? Because I'm getting the same result with the implementation in my app and test it in the provided test app
![image](https://github.com/TorryDo/Floating-Bubble-View/assets/23272313/8c386eae-aefa-4438-a1d8-b69e5c8780d4)

**Comment by dofire** [comment]
My poco device (MIUI) doesn't have this bug, however you can find more information [here](https://stackoverflow.com/questions/62696305/theme-problem-when-dark-mode-is-activated-under-miui-11-12)
